README.md

```diff
@@ -199,41 +199,44 @@ Nevertheless, the decorator should not be applied when it is not necessary becau
+| Angular            | `@angular-eslint/template/no-duplicate-attributes`                                 | https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/docs/rules/no-duplicate-attributes.md                    | ✅                              | ✅                           |                               |                          |
+| Angular            | `@angular-eslint/template/no-positive-tabindex`                                    | https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/docs/rules/no-positive-tabindex.md                       | ✅                              | ✅                           |                               |                          |
+| Angular            | `@angular-eslint/template/use-track-by-function`                                   | https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/docs/rules/use-track-by-function.md                      | ✅                              | ✅                           |                               |                          |
+| Cypress            | All recommended rules from `cypress`                                               | https://github.com/cypress-io/eslint-plugin-cypress#rules                                                                                             | ✅                              |                              | ✅                            |                          |
+| eslint-comments    | All recommended rules from `eslint-comments`                                       | https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/                                                                                     | ✅                              | ✅                           | ✅                            | ✅                       |
+| eslint-comments    | `eslint-comments/require-description`                                              | https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/require-description.html                                                             | ✅                              | ✅                           | ✅                            | ✅                       |
+| eslint-comments    | `eslint-comments/disable-enable-pair` (custom config to allow whole-file disables) | https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/disable-enable-pair.html                                                             | ✅                              | ✅                           | ✅                            | ✅                       |
```

Comment:
The spacing changed so it looks like I changed the entire table, but the only meaningful change to this file is the addition of these rows